model_info/ukwiki.articletopic.md

```diff
@@ -0,0 +1,961 @@
+		Culture.Media.Books                          0.733
+		Culture.Media.Entertainment                  0.256
+		Culture.Media.Films                          0.888
+		Culture.Media.Media*                         0.854
+		Culture.Media.Music                          0.841
+		Culture.Media.Radio                          0.284
+		Culture.Media.Software                       0.243
+		Culture.Media.Television                     0.772
+		Culture.Media.Video games                    0.876
+		Culture.Performing arts                      0.473
+		Culture.Philosophy and religion              0.427
+		Culture.Sports                               0.948
+		Culture.Visual arts.Architecture             0.665
+		Culture.Visual arts.Comics and Anime         0.7
+		Culture.Visual arts.Fashion                  0.372
+		Culture.Visual arts.Visual arts*             0.703
+		Geography.Geographical                       0.669
+		Geography.Regions.Africa.Africa*             0.614
+		Geography.Regions.Africa.Central Africa      0.315
+		Geography.Regions.Africa.Eastern Africa      0.487
+		Geography.Regions.Africa.Northern Africa     0.327
+		Geography.Regions.Africa.Southern Africa     0.39
+		Geography.Regions.Africa.Western Africa      0.281
+		Geography.Regions.Americas.Central America   0.592
+		Geography.Regions.Americas.North America     0.744
+		Geography.Regions.Americas.South America     0.73
+		Geography.Regions.Asia.Asia*                 0.794
+		Geography.Regions.Asia.Central Asia          0.33
+		Geography.Regions.Asia.East Asia             0.771
+		Geography.Regions.Asia.North Asia            0.436
+		Geography.Regions.Asia.South Asia            0.872
+		Geography.Regions.Asia.Southeast Asia        0.705
+		Geography.Regions.Asia.West Asia             0.769
+		Geography.Regions.Europe.Eastern Europe      0.677
+		Geography.Regions.Europe.Europe*             0.758
+		Geography.Regions.Europe.Northern Europe     0.714
+		Geography.Regions.Europe.Southern Europe     0.708
+		Geography.Regions.Europe.Western Europe      0.723
+		Geography.Regions.Oceania                    0.82
+		History and Society.Business and economics   0.458
+		History and Society.Education                0.558
+		History and Society.History                  0.375
+		History and Society.Military and warfare     0.626
+		History and Society.Politics and government  0.61
+		History and Society.Society                  0.238
+		History and Society.Transportation           0.898
+		STEM.Biology                                 0.9
+		STEM.Chemistry                               0.429
+		STEM.Computing                               0.357
+		STEM.Earth and environment                   0.545
+		STEM.Engineering                             0.713
+		STEM.Libraries & Information                 0.343
+		STEM.Mathematics                             0.338
+		STEM.Medicine & Health                       0.596
+		STEM.Physics                                 0.182
```

Comment:
This is the worst label.  You'd think we'd be good at it!

Reply:
Just checked and many of the other models do poorly here as well.  I wonder if our labels for physics are a bit messy.

Reply:
Perhaps. Now that you are around a bit. Can I upload the other models. I've built them. Just so you can check the fitness.

Reply:
Sure!  I'll take a look and comment.  Feel free to ping me directly if you want me to look at something :)  I don't mind.

Reply:
Yaaayyyy. I just pushed. The pr_auc values for the different wikis are the parts I'm concerned about haha.